package-lock.json

```diff
@@ -442,6 +443,13 @@
         "@types/node": "*"
       }
     },
+    "node_modules/@types/w3c-direct-sockets": {
+      "version": "0.0.2",
+      "resolved": "https://us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/@types/w3c-direct-sockets/-/w3c-direct-sockets-0.0.2.tgz",
```

Comment:
This line is the problem. It's telling npm to download the package from a URL which is not publicly accessible.

Reply:
I had something wrong with my local `.npmrc` file and registry was set to Google internal. Now it should work!